Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:lts
```

Comment:
This should be pinned to a specific node version, not just LTS.

Reply:
Alright so should I pin it to the LTS version as of the current date?